client/src/components/toPayItem.js

```diff
@@ -61,20 +75,20 @@ export default class ToPayItem extends React.Component {
             innerDivStyle ={ListItemStyle}
             secondaryText ={this.props.message}
             primaryText ={'$' + this.props.amount.toFixed(2)}
-            rightIconButton={<IconButton><CommunicationEmail/></IconButton>}
+            rightIconButton={<IconButton onTouchTap={this.handleMsgButton}><CommunicationEmail/></IconButton>}
             secondaryTextLines={1}
           />
         </div>
         <div style={cellDivStyle}>
-          <RaisedButton style={ButtonStyle} label={'Pay'}/>
+          <RaisedButton style={ButtonStyle} label={'Pay'} backgroundColor={red} labelColor={white}/>
         </div>
       </div>
     );
   };
 }
 
 ToPayItem.propTypes = {
-  message: PropTypes.string,
-  amount: PropTypes.number,
+  message: React.PropTypes.string,
```

Comment:
Please remove `React.PropTypes`, and use the import version of `PropTypes` to reduce the warning noise.